x/pairing/keeper/unstaking_test.go

```diff
@@ -48,3 +48,39 @@ func TestUnstakeAndSlashProposal(t *testing.T) {
 		}
 	}
 }
+
+// TestVaultOperatorUnstakeAndSlashProposal tests that when running a proposal, both the operator
+// and vault addresses should work
+// Scenarios:
+// 1. simulate unstake proposal with vault address -> should work
+// 2. simulate with operator -> should work
+func TestVaultOperatorUnstakeAndSlashProposal(t *testing.T) {
+	ts := newTester(t)
+	ts.setupForPayments(1, 0, 0)
+
+	acc, _ := ts.GetAccount(common.PROVIDER, 0)
+	operator := acc.Addr.String()
+	vault := acc.Vault.Addr.String()
+
+	tests := []struct {
+		name    string
+		creator string
+		valid   bool
```

Comment:
Redundant in this test

Reply:
done. see https://github.com/lavanet/lava/commit/88b007bc6f553ad6d48b971c823b1f0f4cb3738b